sdk/dotnet/Pulumi/Resources/InvokeArgs.cs

```diff
@@ -13,10 +13,6 @@ public abstract class InvokeArgs : InputArgs
 
         private protected override void ValidateMember(Type memberType, string fullName)
```

Comment:
I'd have to double check, but after removing the implementation below, I suspect no subclass actually has an implementation and `ValidateMember` could be nuked entirely. But that can be cleaned up separately.

Reply:
I'll try that.